spec/spec_helper.rb

```diff
@@ -31,6 +31,9 @@ def image_path(path, options = {})
 Dir[Rails.root.join("spec/shared/**/*.rb")].each { |f| require f }
 # include the manageiq-gems-pending matchers
 Dir[ManageIQ::Gems::Pending.root.join("spec/support/custom_matchers/*.rb")].each { |f| require f }
+Vmdb::Plugins.instance.vmdb_plugins.each do |plugin|
+  Dir[plugin.root.join("spec/support/**/*.rb")].each { |f| require f }
+end
```

Comment:
I don't like this.  The core shouldn't load all of the plugin's support dirs because we don't run their tests, except in the specific case of the shared examples for Tower.